frontend/src/container/LogsError/LogsError.tsx

```diff
@@ -7,14 +7,29 @@ import history from 'lib/history';
 import { ArrowRight } from 'lucide-react';
 import { isCloudUser } from 'utils/app';
 
-export default function LogsError(): JSX.Element {
+export interface QueryRangeErrorCauseType {
+	error: string;
+}
+
+export default function LogsError({
+	errorData,
+}: {
+	errorData?: Error | null;
+}): JSX.Element {
 	const handleContactSupport = (): void => {
 		if (isCloudUser()) {
 			history.push('/support');
 		} else {
 			window.open('https://signoz.io/slack', '_blank');
 		}
 	};
+
+	let serverErrorMessage = null;
+
+	if (errorData && errorData.cause) {
+		serverErrorMessage = (errorData.cause as QueryRangeErrorCauseType).error;
```

Comment:
there should not be any type of assertion bad practice 

ideally, this should be handled from central place

Reply:
Error type has a standard type in case of New Error() and the cause property type is set to unknown and we are setting cause to be an object with the error property. Hence this.